core/pom.xml

```diff
@@ -413,6 +413,13 @@ THE SOFTWARE.
       <artifactId>jline</artifactId>
       <version>0.9.94</version>
```

Comment:
infact the [version of groovy](https://repo1.maven.org/maven2/org/codehaus/groovy/groovy-all/1.8.9/groovy-all-1.8.9.pom) we use uses 1.0!